packages/drivers/routerlicious-driver/src/deltaStorageService.ts

```diff
@@ -115,14 +116,26 @@ export class DeltaStorageService implements IDeltaStorageService {
 			async (event) => {
 				const restWrapper = await this.getRestWrapper();
 				const url = this.getDeltaStorageUrl();
-				const response = await restWrapper.get<ISequencedDocumentMessage[]>(url, {
-					from: from - 1,
-					to,
-				});
+				const response = await restWrapper.get<IR11sResponse<ISequencedDocumentMessage[]>>(
+					url,
+					{
+						from: from - 1,
+						to,
+					},
+					undefined,
+					true,
+				);
 				event.end({
-					count: response.length,
+					count: response.content.length,
```

Comment:
I think in other places we use `length`. If yes, let's change it to be consistent across the board.